.ci/release-settings.xml

```diff
@@ -0,0 +1,29 @@
+<settings>
+  <servers>
+    <server>
+      <id>sonatype-nexus-staging</id>
+      <username>SONATYPE_USER</username>
+      <password>SONATYPE_PWD</password>
+    </server>
+    <server>
+      <id>sourceforge-new-site</id>
```

Comment:
please remove this server.
It is old, I double checked, we do not use it anymore.
I updated wiki also.

Reply:
Changes Done.